### ISSUE TITLE
E1305 false positive when using nested replacement fields in str formatting

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#294]
Originally reported by: **Chris Rebert (BitBucket: [cvrebert](http://bitbucket.org/cvrebert), GitHub: @cvrebert?)**

---

```
#!bash

$ pylint --version
No config file found, using default configuration
pylint 1.3.0, 
astroid 1.2.0, common 0.62.0
Python 2.7.7 (default, Jun 14 2014, 23:12:13) 
[GCC 4.2.1 Compatible Apple LLVM 5.1 (clang-503.0.40)]
```

```
#!python

# foobarbaz.py
def prnt(msg, width):
    return '{0:>{1}}'.format(msg, width)
```

```
#!bash

$ pylint -r n -f parseable foobarbaz.py
No config file found, using default configuration
************* Module foobarbaz
foobarbaz.py:2: [E1305(too-many-format-args), prnt] Too many arguments for format string
```

```
#!python

Python 2.7.7 (default, Jun 14 2014, 23:12:13) 
[GCC 4.2.1 Compatible Apple LLVM 5.1 (clang-503.0.40)] on darwin
>>> from foobarbaz import prnt
>>> prnt("Hello", 10)
'     Hello'
>>> prnt("Hello", 15)
'          Hello'
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/294
